com.unity.netcode.adapter.utp/Tests/Runtime/TransportTests.cs

```diff
@@ -305,6 +267,74 @@ public IEnumerator ReceiveMultipleClients([ValueSource("k_DeliveryParameters")]
 
             yield return null;
         }
+
+        // Check that we get disconnected when overflowing the reliable send queue.
+        [UnityTest]
+        public IEnumerator DisconnectOnReliableSendQueueOverflow()
+        {
+            InitializeTransport(out m_Server, out m_ServerEvents);
+            InitializeTransport(out m_Client1, out m_Client1Events);
+
+            m_Server.StartServer();
+            m_Client1.StartClient();
+
+            yield return WaitForNetworkEvent(NetworkEvent.Connect, m_Client1Events);
+
+            m_Server.Shutdown();
+
+            var numSends = (UnityTransport.InitialMaxSendQueueSize / 1024) + 1;
+
+            for (int i = 0; i < numSends; i++)
+            {
+                var payload = new ArraySegment<byte>(new byte[1024]);
+                m_Client1.Send(m_Client1.ServerClientId, payload, NetworkDelivery.Reliable);
+            }
+
+            LogAssert.Expect(LogType.Error, "Couldn't add payload of size 1024 to reliable send queue. " +
+                "Closing connection 4294967296 as reliability guarantees can't be maintained. " +
+                "Perhaps 'Max Send Queue Size' (98304) is too small for workload.");
```

Comment:
can we get `m_MaxSendQueueSize` variable here somehow instead of hardcoding `98304` inline?
(maybe `InternalsVisibleTo:` stuff?)

Reply:
Good point. I'll use `InitialMaxSendQueueSize` which is public and already used above to determine how much data to send to overflow the receive queue.